auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -332,12 +350,14 @@ public String toString() {
             return "IdpConfig{" +
                     "mProviderId='" + mProviderId + '\'' +
                     ", mScopes=" + mScopes +
+                    ", mParams=" + mParams.toString() +
```

Comment:
One last nit! 😄 We don't need the `toString()` call, `mParams` alone is just fine.

Reply:
Removed.